content/docs/command-reference/fetch.md

```diff
@@ -51,6 +46,15 @@ project's cache                  ++ | dvc pull |
  workspace
 ```
 
+Without arguments, it downloads all files and directories missing from the
+project, found as <abbr>outputs</abbr> in all `dvc.yaml` and `dvc.lock` pairs,
```

Comment:
pairs -> files? (too complicated, introduces new concept)

if it relies on `dvc.yaml` then it's fine to omit `dvc.lock` here -it's an implementation detail (can be described if describe how it works). Or mentioned as a separate sentence.

Reply:
- [x] pairs -> files
- [x] fine to omit dvc.lock here

Reply:
> fine to omit dvc.lock here -it's an implementation detail (can be described if describe how it works

Yeah this is a widespread problem in some cmd refs. Too many or too few mentions of dvc.lock and possibly in the wrong ways...